near-campaign/src/add_keys.rs

```diff
@@ -4,11 +4,12 @@ use crate::*;
 impl Campaign {
   #[private]
   pub fn add_keys(&mut self, keys: Vec<Base58PublicKey>) {
+    // TODO this doesn't handle overlap of keys that already exist in the map?
```

Comment:
Yeah, this is an open question - do we need to check that the key is already in the state? The current workflow - we only call this method multiple times during the campaign creations.